docs/web-wallet.md

```diff
@@ -0,0 +1,254 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The Web Wallet is a web tool meticulously developed to streamline the verification process of credentials. Built for verifiers, it seamlessly integrates into your applications, enhancing the user experience by facilitating the request and verification of credentials within a unified and simplified workflow.
+
+## Features
+
+- **Seamless Authentication and Credential Management**: This tool enables users to authenticate via their Ethereum wallets, and automatically creates a derived identity wallet. Credentials undergo encryption using keys and are securely stored in end-to-end encrypted cloud storage*, facilitating effortless access across multiple devices while streamlining processes.
+
+:::note
+
+*The Cloud Storage serves as a secure repository for encrypted credentials, offering users the convenience of multi-device access to credentials and serving as a reliable backup solution. When a user signs the message using their Ethereum private key, an identity is derived and storage keys are generated. The digital signature scheme employed is ed25519.  All the documents stored in the storage are end-to-end encrypted using AES256-GCM, guaranteeing that only the user with the storage keys can access his credentials, reinforcing the security of the stored information.
+
+:::
+
+- **Enhanced User Experience**: Prioritizing simplicity, the Web Wallet presents a user-friendly interface. From identity creation to credential claiming and verification, all steps seamlessly integrate, minimizing complexity.
+- **Embedded Issuance**: The Web Wallet provides embedded issuance, allowing users to claim their credentials within the verification flow if they haven't done so before.
+
+:::note
+
+Currently, the tool supports embedded issuance for only two types of credentials: 
+- Proof of Identity (refer to the schema [here](https://github.com/anima-protocol/claims-polygonid/blob/main/schemas/json-ld/poi-v1.json-ld))
+- Proof of Life (refer to the schema [here](https://github.com/anima-protocol/claims-polygonid/blob/main/schemas/json-ld/pol-v1.json-ld))
+
+The tool still supports the verification of other credentials, which can be issued on the Polygon ID Wallet App.
+:::
+- **Simplified Integration**: Integration into your application is effortless, reducing the development workload and ensuring swift implementation.
+- **On-Chain and Off-Chain Query Verification**: Supporting both on-chain and off-chain query verification, the Web Wallet offers flexibility to meet diverse verification needs.
+- **Consistency and Accuracy**: With a standardized interface, the tool promotes uniformity across users and organizations, minimizing errors.
+
+## Getting Started
+
+Learn more about the Web Wallet in the video below:
+
+<div align="center" style={{margin: 40}}>
+<iframe src="https://www.youtube.com/embed/" width="100%" height="450" allowfullscreen></iframe>
```

Comment:
Since `src` is not yet filled, I would mark this pull request as a draft so we don't forget and merge it by mistake.

Reply:
Sure , makes sense @amonsosanz

Reply:
Have removed it for now so that this goes to staging- i will add the link when ready

Reply:
But this is not JS so `//add yt video` is rendered...

Reply:
Fixed